documentation/Troubleshooting.md

```diff
@@ -1427,6 +1428,40 @@ Consider adjusting the buffer size in the `Audit` daemon configuration to avoid
 
 > **Note**: Not applicable.
 
+## Calico Generates High Amount of Logs and Consumes a lot of CPU
+
+### Description
+Calico-node pods generate a lot of logs and consume a lot of resources that causes pod restart. Such logs can be found in calico-node pods:
+
+```bash
+[WARNING][89] felix/int_dataplane.go 1822: failed to wipe the XDP state error=failed to load BPF program (/usr/lib/calico/bpf/filter.o): stat /sys/fs/bpf/calico/xdp/prefilter_v1_calico_tmp_A: no such file or directory
+libbpf: Error loading BTF: Invalid argument(22)
+```
+
+### Alerts
+Not applicable.
+
+### Stack trace(s)
+Not applicable.
+
+### How to solve
+As WA XDP acceleration can be turned off by adding the following parameter:
+
+```bash
+ # kubectl -n kube-system edit ds calico-node
+...
+spec:
+  template:
+    spec:
+      containers:
+      - env:
+...
+        - name: FELIX_XDPENABLED
+          value: "false"
+... 
+```
+Pods should stop generating such amount of logs and resource consumption should normalize.
```

Comment:
Can we apply this using KubeMarine calico plugin configuration/upgrade? Such manual edit may be lost on calico upgrade

Reply:
@theboringstuff 
WA described is verified and it works.
We can apply it using KubeMarine I believe, but it is not reproducible, so I'm not sure.

Reply:
@theboringstuff 
KubeMarine approach has been added.
Please review.

Reply:
Looks good to me